apps/mobile/src/components/ContextSwitchMenu/ContextSwitchMenu.js

```diff
@@ -0,0 +1,137 @@
+import { map, sortBy } from 'lodash/fp'
+import useCurrentUser from '@hylo/hooks/useCurrentUser'
+import useCurrentGroup from '@hylo/hooks/useCurrentGroup'
+import { PUBLIC_GROUP } from '@hylo/presenters/GroupPresenter'
+import useChangeToGroup from 'hooks/useChangeToGroup'
+
+export default function ContextSwitchMenu () {
+  const [{ currentUser }] = useCurrentUser()
+  const [{ currentGroup }] = useCurrentGroup()
+  const changeToGroup = useChangeToGroup()
+  const myGroups = [PUBLIC_GROUP].concat(sortBy('name', map(m => m.group, currentUser.memberships)))
+
+  return (
+    <Animated.View className='flex-col h-full bg-theme-background z-50 items-center py-2 px-3'>
+      {/* FlatList used instead of FlashList because of strict-sizing requirements of FlashList */}
+      <FlatList
+        data={myGroups}
+        keyExtractor={(item) => item.id}
+        renderItem={({ item }) => (
+          <NavRow
+            changeToGroup={changeToGroup}
+            item={item}
+            currentGroupSlug={currentGroup?.slug}
+          />
+        )}
+        showsVerticalScrollIndicator={false}
+      />
+      <View className='w-full mt-auto bg-theme-background pt-4'>
+        <TouchableOpacity 
+          onPress={() => {}} // TODO redesign: Needs to open some creation dialog...
+          style={styles.rowTouchable}
+          activeOpacity={0.7}
+        >
+          <View
+            className={`
+                bg-primary relative flex flex-col text-primary-foreground items-center justify-center
+                w-14 h-14 min-h-10 rounded-lg drop-shadow-md opacity-60 scale-90
+            `}
+          >
+            <Plus />
+          </View>
+        </TouchableOpacity>
+        <TouchableOpacity
+          onPress={() => {}} // Needs to open some creation dialog?
+          style={styles.rowTouchable}
+          activeOpacity={0.7}
+        >
+          <View className='bg-primary relative flex flex-col text-primary-foreground items-center justify-center w-14 h-14 min-h-10 rounded-lg drop-shadow-md opacity-60 scale-90'>
+            <CircleHelp />
+          </View>
+        </TouchableOpacity>
+      </View>
+    </Animated.View>
+  )
+}
+
+function NavRow ({ item, changeToGroup, currentGroupSlug, badgeCount = 0, className }) {
+  const { id, avatarUrl, name, slug } = item
+  const newPostCount = Math.min(99, item.newPostCount)
+  const highlight = slug === currentGroupSlug
+
+  return (
+    <TouchableOpacity
+      key={id} 
+      onPress={() => changeToGroup(item?.slug, false)}
+      style={styles.rowTouchable}
+      activeOpacity={0.7}
+    >
+      <View
+        className={[
+          'bg-primary relative flex flex-col items-center justify-center w-14 h-14 min-h-10 rounded-lg drop-shadow-md opacity-60 scale-90',
+          highlight && 'border-3 border-secondary opacity-100 scale-100',
+          badgeCount > 0 && 'border-3 border-accent opacity-100 scale-100',
+          className
+        ].filter(Boolean).join(' ')}
```

Comment:
Please go ahead add clsx again and apply here and in similar cases. Didn't intend to veto it for good, was just trying to understand where we were headed in the earlier PR where it came up.